ci/docker/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
   centos7_cpu:
     # The resulting image will be named build.centos7_cpu:latest and will be
     # pushed to mxnetci under this name
-    image: build.centos7_cpu:latest
+    image: mxnetci/build.centos7_cpu:latest
```

Comment:
Can we parametrize this? This code is not testable in another environment

Reply:
I don't think that's possible

Reply:
https://docs.docker.com/compose/compose-file/#variable-substitution